src/chttpd/src/chttpd.erl

```diff
@@ -802,7 +802,10 @@ absolute_uri(#httpd{absolute_uri = URI}, Path) ->
     URI ++ Path.
 
 unquote(UrlEncodedString) ->
-    mochiweb_util:unquote(UrlEncodedString).
+    case chttpd_util:get_chttpd_config_boolean("decode_plus_to_space", false) of
```

Comment:
`chttpd_util:get_chttpd_config_boolean` will fallback to look in httpd section as well. We probably don't care about that as it's a new setting not one which moved from httpd section to chttpd.